src/Cronofy/AddToCalendarRequestBuilder.cs

```diff
@@ -0,0 +1,82 @@
+﻿namespace Cronofy
+{
+    using System;
+    using Cronofy.Requests;
+
+    /// <summary>
+    /// Class to build an add to calendar request.
+    /// </summary>
+    public sealed class AddToCalendarRequestBuilder : IBuilder<AddToCalendarRequest>
+    {
+        /// <summary>
+        /// The oauth details for the request.
+        /// </summary>
+        private AddToCalendarRequest.OAuthDetails oauth;
+
+        /// <summary>
+        /// The event details for the request.
+        /// </summary>
+        private UpsertEventRequest @event;
```

Comment:
Its a shame event is a keyword, though I can't think of a better name at the moment